### ISSUE TITLE
Diablo 3 No Longer Loads

### ISSUE THREAD

**Issue by thefooz** [yuzu-emu/yuzu#6058]
At some point during the development process, it appears that Diablo 3, a game that previously loaded and played perfectly, stopped working. It previously ran in OpenGL and Vulkan, now it won't even open.

**Comment by michaelpegaso** [comment]
The same thing happens to me for some versions

**Comment by fuhuan416** [comment]
Yeah, this is a problem that has existed for a long time. I use EA-1152 to start 100% successfully. Once I use this version, it takes luck to start successfully, even EA-1519.

**Comment by gandalfthewhite19890404** [comment]
Mainline 567 works good, 569 - may boots, may does not (without checkbox "Use assembly shaders" boots almost always).

**Comment by FearlessTobi** [comment]
Could you please provide a log file of trying to boot the game on the latest yuzu version?

**Comment by gandalfthewhite19890404** [comment]
Latest mainline. Boots but at 2nd try.
Normal GPU accuracy, disabled async shader compilation, disabled assembly shaders.
[yuzu_log.zip](https://github.com/yuzu-emu/yuzu/files/6210012/yuzu_log.zip)

**Comment by gandalfthewhite19890404** [comment]
For me boots only with OpenGL, with some trick - go to settings, change graphic API to Vulkan and then to openGL immediately.
